src/llnode.cc

```diff
@@ -135,6 +135,29 @@ bool SetPropertyColorCmd::DoExecute(SBDebugger d, char** cmd,
   return false;
 }
 
+bool SetTreePaddingCmd::DoExecute(SBDebugger d, char** cmd,
+                                  SBCommandReturnObject& result) {
+
+  if (cmd == nullptr || *cmd == nullptr) {
+    result.SetError("USAGE: v8 settings set tree-padding [1..10]");
+    return false;
+  }
+  Settings* settings = Settings::GetSettings();
+  std::stringstream option(cmd[0]);
+  int padding;
+
+  if (!(option >> padding)) {
```

Comment:
Add a comment explaining this converts string to int and returns false if it can't.

Reply:
Added an explanation